site/docs/numbers/porting/csrLookupTool.mdx

```diff
@@ -32,47 +32,18 @@ This walks through how to programmatically lookup Customer Service Records (CSRs
 
```

Comment:
Change "(CSRs for short)" to just "(CSRs)
Change "Phone Numbers" to "phone numbers"